### ISSUE TITLE
GUI tools in the macOS nightly package are broken

### ISSUE THREAD

**Issue by tneele** [mCRL2org/mCRL2#1552]
When running one of the GUI tools from the nightly package, it throws the following error:
```
objc[830]: Class QMacAutoReleasePoolTracker is implemented in both /Applications/mCRL2.app/Contents/Frameworks/QtCore.framework/Versions/5/QtCore (0x10f3665a0) and /Applications/mCRL2.app/Contents/lib/QtCore.framework/Versions/5/QtCore (0x10d8405a0). One of the two will be used. Which one is undefined.
objc[830]: Class QT_ROOT_LEVEL_POOL__THESE_OBJECTS_WILL_BE_RELEASED_WHEN_QAPP_GOES_OUT_OF_SCOPE is implemented in both /Applications/mCRL2.app/Contents/Frameworks/QtCore.framework/Versions/5/QtCore (0x10f366618) and /Applications/mCRL2.app/Contents/lib/QtCore.framework/Versions/5/QtCore (0x10d840618). One of the two will be used. Which one is undefined.
objc[830]: Class RunLoopModeTracker is implemented in both /Applications/mCRL2.app/Contents/Frameworks/QtCore.framework/Versions/5/QtCore (0x10f366640) and /Applications/mCRL2.app/Contents/lib/QtCore.framework/Versions/5/QtCore (0x10d840640). One of the two will be used. Which one is undefined.
qt.qpa.plugin: Could not load the Qt platform plugin "cocoa" in "" even though it was found.
This application failed to start because no Qt platform plugin could be initialized. Reinstalling the application may fix this problem.

Available platform plugins are: cocoa, minimal, offscreen, webgl.
```

**Comment by tneele** [comment]
I tried to investigate this, but I have not been able to resolve this problem so far. The fact that I don't own a Mac means that I cannot debug this effectively.

This issue is related to the snippet below, which causes the Qt libraries to be copied to the app bundle twice.
https://github.com/mCRL2org/mCRL2/blob/2208327f2f551fbc03fcd02f0da68b6ee39a1f41/build/packaging/CMakeLists.txt#L76-L97
Removing these lines leads to the GUI tools not being able to find the libraries at all. The problems were probably introduced after upgrading the build server to Qt 5.13.0.

**Comment by jgroote** [comment]
I just investigated this by downloading the nightly built for the mac.
In my variant there is no directory mCRL2.app/Contents/Frameworks/
although the error message clearly indicate that this directory should exist.
Something fishy is going on here. Was there some odd overwriting of an
existing mCRL2.app file??

**Comment by tneele** [comment]
We are investigating this right now and we think we have a solution. I will commit it shortly.

**Comment by tneele** [comment]
Fixed in commit 1a47a6561d9acb5d1336ab817ffacf7773941c4b.
